imports/include/niTClk.h

```diff
@@ -0,0 +1,274 @@
+// ***************************************************************************************
+//    NI-TClk
+//
+//    National Instruments library for TClk synchronization
+//
+//    Copyright (c) National Instruments 2003.  All Rights Reserved.
```

Comment:
In latest release, copyright will be 2003-2021. You can resolve this comment after using files from latest released version

Reply:
Done.